CheatSheets/html-cheatsheet.md

```diff
@@ -67,15 +64,34 @@
 | `<footer> ... </footer>`         | Defines a footer for a document or section                                                  |
 | `<header> ... </header>`         | Specifies a header for a document or section                                                |
 | `<main> ... </main>`             | Specifies the main content of a document                                                    |
-| `<mark> ... </mark>`             | Defines marked/highlighted text                                                             |
 | `<nav> ... </nav>`               | Defines navigation links                                                                    |
 | `<section> ... </section>`       | Defines a section in a document                                                             |
 | `<summary> ... </summary>`       | Defines a visible heading for a `<details>` element                                         |
 | `<time> ... </time>`             | Defines a date/time                                                                         |
 
 **[🔼Back to Top](#table-of-contents)**
 
+## Formatting
+>Formatting elements were designed to display special types of text:
+
+| Command                  | Description                                          |
+| ------------------------ | ---------------------------------------------------- |
+| `<b> ... </b>`           | Defines bold text                                    |
+| `<em> ... </em>`         | Defines emphasized text                              |
+| `<i> ... </i>`           | Defines a part of text in an alternate voice or mood |
+| `<small> ... </small>`   | Defines smaller text                                 |
+| `<strong> ... </strong>` | Defines important text                               |
+| `<sub> ... </sub>`       | Defines subscripted text                             |
+| `<sup> ... </sup>`       | Defines superscripted text                           |
```

Comment:
There are some `duplicate tags`. Please, `rearrange and delete` the duplicate tags. Your Formatting section is looking very good. If those tags already exist, then remove the older one and rearrange them like the `Formatting section`. Try to check your newly added tags with the original [html-cheatsheet.md](https://github.com/crescentpartha/CheatSheets-for-Developers/blob/main/CheatSheets/html-cheatsheet.md) file. Otherwise, creating a new conflict again.

Reply:
I already removed older repeated tags please check